apps/mobile/src/navigation/linking/index.js

```diff
@@ -36,12 +36,16 @@ export const routingConfig = {
 
   '/hylo-editor':                                            `${AUTH_ROOT_SCREEN_NAME}/HyloEditor`,
 
-  // context group routes (/all, /public)
-  '/:groupSlug(all|public)':                                 `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab`,
+  // context group routes (/all, /public, /my)
```

Comment:
@thomasgwatson you had a typo in the screen path here that I fixed, but I also removed your comment question about whether the screen path should "just include /Stream?" (paraphrase) The answer to that is, sure, if it is what is making sense now. I had let "Home Tab" handle what to do with it before (so basically sending it to the default which would have been Group Navigation which would have switched to the context and then forwarded to Stream. It will be a little different now, so I don't know the answer. But if you want /all and /public to go directly to the Steram for now, then this is the correct configuration.